crates/qt-build-utils/src/lib.rs

```diff
@@ -15,6 +15,11 @@
 
 #![allow(clippy::too_many_arguments)]
 
+mod builder;
```

Comment:
maybe these are more generators than builders as they are creating strings rather than building anything :thinking:

Reply:
Hm, most of these are just for generating QML-specific things, so why not split it into a `qml` and a `qrc` module?